dev-lang/luajit/luajit-2.1.0_beta3_p20220613.ebuild

```diff
@@ -31,6 +31,21 @@ IUSE="lua52compat static-libs"
 
 S="${WORKDIR}/LuaJIT-${GIT_COMMIT}"
 
+src_configure() {
+	tc-export_build_env
+
+	# You need to use a 32-bit toolchain to build for a 32-bit architecture.
+	# Some 64-bit toolchains (like amd64 and ppc64) usually have multilib
+	# enabled, allowing you to build in 32-bit with -m32. This won't work in all
+	# cases, but it will otherwise just break, so it's worth trying anyway. If
+	# you're trying to build for 64-bit from 32-bit, then you're screwed, sorry.
+	if tc-is-cross-compiler && [[ $($(tc-getBUILD_CPP) -P - <<< __INTPTR_WIDTH__) = 64 &&
+	                              $($(tc-getCPP)       -P - <<< __INTPTR_WIDTH__) = 32 ]]; then
```

Comment:
Let's do == for both tests given we're in Bash.

Reply:
Done. I also decided to check with `!=` for CBUILD. 128-bit will totally be a thing. :stuck_out_tongue: